lib/ProductOpener/Import.pm

```diff
@@ -2772,10 +2772,26 @@ sub import_csv_file ($args_ref) {
 	}
 
 	# sync CRM
+	my @csv_from_sftp_dir = qw(
+		https://www.carrefour.fr
+		https://www.intermarche.com/
+	);
+
 	foreach my $org_id (keys %{$stats_ref->{orgs_existing}}) {
 		update_import_date($org_id, $time);
 		if (exists $args_ref->{source_id}) {
-			update_last_import_type($org_id, uc($args_ref->{source_id}));
+			if ($args_ref->{source_id} eq 'agena3000') {
```

Comment:
Can you add Bayard there? It's similar to agena3000 and equadis

Reply:
I add the GS1 tag ? @stephanegigandet